### ISSUE TITLE
@jupyterlab/hub-extension in jupyterlab==3.0.0-rc.6 not appearing in the menus/commands

### ISSUE THREAD

**Issue by snickell** [jupyterlab/jupyterlab#9428]
Testing with `jupyterlab==3.0.0-rc.6` (we can't use 3.0.0-rc.11 atm because react 0.17 is incompatible with libraries used by some of our extensions), we're not finding the `Hub Control Panel` and cousins (any of the commands added in: https://github.com/jupyterlab/jupyterlab/blob/master/packages/hub-extension/src/index.ts) in the menus or the commands.

**Comment by snickell** [comment]
This is tested running on `jupyterhub==1.2.2`, as distributed in the z2jh kubernetes-based juphub docker image `docker.io/jupyterhub/k8s-hub:0.10.6-n000.hed9b2f4f`, also replicated on a direct system install of `jupyterhub==1.2.2` on Ubuntu running jupyterlab with dockerspawner.

**Comment by cnydw** [comment]
This issue has been fixed by https://github.com/jupyterlab/jupyterlab/pull/9256 , available from `jupyterlab==3.0.0-rc.7`

**Comment by costrouc** [comment]
I am running into the same issue with version 3.0.3

![image](https://user-images.githubusercontent.com/1740337/104334795-cd50a300-54c0-11eb-949a-ff22e3634b05.png)

![image](https://user-images.githubusercontent.com/1740337/104334846-d8a3ce80-54c0-11eb-97c2-7be2befb91df.png)

Jupyterlab was launched via jupyterhub/batchspawner via the following command


```shell
batchspawner-singleuser jupyterhub-singleuser --ip=0.0.0.0 --SingleUserNotebookApp.default_url=/lab
```

When I pin the jupyterlab version to 2.2.9 the menu hub and logout show up

**Comment by blink1073** [comment]
Fixed by #9604, will cut a release today.

**Comment by costrouc** [comment]
Can confirm that 3.0.4 resolved the issue.
